tutorials/step_by_step/animations.rst

```diff
@@ -47,8 +47,8 @@ for similar functionality.)
 
 .. image:: /img/animeditor.png
 
-But the second, and most important, is that the property editor enters
-into "animation editing" mode. In this mode, a key icon appears next to
+The second and most important, is that the property editor enters
```

Comment:
If you remove the first comma, you need to remove the second. Now the sentence is cut in a grammatically incorrect way.